activetable.js

```diff
@@ -107,15 +128,45 @@ var ActiveTable = (function () {
             return $row;
         }
 
+        function makeHelp(help_text) {
+            var help_active = false;
+            if (!help_text) return;
+            $('#help-text').text(help_text);
+            $('#help-button').click(function(e) {
+                $('#help-text').toggle();
+                help_active = !help_active;
+                $(this).text(help_active ? '-help' : '+help');
+            });
+            $('#help').show();  // Show the div with the button, not the text.
+        }
+
+        function makeColGroup(num_cols, column_widths) {
+            var w;
+            for (var i = 0; i < num_cols; i++) {
+                w = column_widths !== null ? column_widths[i] : 800 / num_cols;
+                $('<col>').css('width', w).appendTo('#activeTable colgroup');
+            }
+        }
+
+        function setRowHeight(row_height) {
+            $('#row-height-style').text(
+                'tr { height: ' + row_height + 'px; }\n' +
+                'input { height: ' + (row_height - 2) + 'px; }\n'
+            );
```

Comment:
@e-kolpakov I thnk this is a pretty standard way of injecting styles for classes, and I don't know any other way of doing it.  I tested it on one version each of Safari, IE, Chrome and Firefox, and it works fine.